packager/media/trick_play/trick_play_handler_unittest.cc

```diff
@@ -58,7 +58,8 @@ class TrickPlayHandlerTest : public MediaHandlerTestBase {
     trick_play_handler_.reset(new TrickPlayHandler(trick_play_options));
     // The output stream size is number of trick play stream + one
     // non-trick-play stream.
-    SetUpGraph(1, trick_play_options.trick_play_rates.size() + 1,
+    SetUpGraph(1,
```

Comment:
Perhaps changing the function signature of SetUpGraph to 

  void SetUpGraph(size_t num_inputs,
                  size_t num_outputs,
                  std::shared_ptr<MediaHandler> handler);